1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md

```diff
@@ -0,0 +1,17 @@
+# रिसर्च सेंसर और एक्चुएटर्स
+
+## निर्देश
+
+इस पाठ में सेंसर और एक्चुएटर्स शामिल थे। अनुसंधान और एक सेंसर और एक एक्चुएटर का वर्णन करें जिसका उपयोग IoT देव किट के साथ किया जा सकता है, जिसमें शामिल हैं:
+
+* यह क्या करता है
+* अंदर इस्तेमाल होने वाले इलेक्ट्रॉनिक्स / हार्डवेयर
+* क्या यह एनालॉग या डिजिटल है
+* इनपुट या माप की इकाइयाँ और रेंज क्या है
+
+## रूब्रिक
+
+| मानदंड | अनुकरणीय | पर्याप्त | सुधार की जरूरत |
+| -------- | ------------- | -------- | ----------------- |
+| एक सेंसर का वर्णन करें | ऊपर सूचीबद्ध सभी 4 अनुभागों के विवरण सहित एक सेंसर का वर्णन किया। | एक सेंसर का वर्णन किया, लेकिन ऊपर के केवल 2-3 अनुभाग प्रदान करने में सक्षम था | एक सेंसर का वर्णन किया है, लेकिन ऊपर दिए गए अनुभागों में से केवल 1 प्रदान करने में सक्षम था |
```

Comment:
can we change these last line's and the last 2nd column lines as it seems a bit confusing? same in line 17

Reply:
Kindly explain.

Reply:
I have explained you in private message